apps/explorer/src/components/module/PkgModulesWrapper.tsx

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { useState, useEffect, useCallback } from 'react';
+import { useSearchParams } from 'react-router-dom';
+
+import ModuleView from './ModuleView';
+
+import styles from './ModuleView.module.css';
+
+import { Tab, TabGroup, TabList, TabPanel, TabPanels } from '~/ui/Tabs';
+
+type Modules = {
+    title: string;
+    content: [moduleName: string, code: string][];
+};
+
+interface Props {
+    id?: string;
+    data: Modules;
```

Comment:
Rather than a `data` prop, let's make the props interface take `title` and `modules` (renamed from `content`) directly.